### DISCUSSION TITLE
Problem with global HDR toggle

### DISCUSSION THREAD

**Discussion by Titanos** [awawa-dev/HyperHDR#630] [Q&A] [answered]
Hi,

i am using HyperHDR on my HDR capable TV and Windows 11 computer, without any usb grabbers. Instead i am using the DirectX 11 screen grabber but i'm having issues with the global HDR toggle.

The global toggle doesn't seem to apply to the DirectX 11 screen grabber "HDR to SDR tone mapping" option. I have tried switching the "Global HDR toggle" and also sending JSON commands but it does not make any difference. Can this option not be changed via global toggle and/ore JSON command?

I have tried both commands, but no success.

{
	"command":"videomodehdr",
	"HDR":0,
	"flatbuffers_user_lut_filename":""
}

{
	"command":"componentstate",
	"componentstate":
	{
		"component":"HDR",
		"state": false
	}
}

**Comment by awawa-dev** [comment]
Hi
Currently software grabber tone mapping is not connected to global HDR component and works as independent one.

**Reply by Titanos** [reply]
Is there any way to control this toggle by JSON command or any other method other than via the web interface?

**Reply by awawa-dev** [reply, marked as answer]
This is no easy way currently, it requires a similar approach to saving settings through the HyperHDR panel, in particular implementing authentication or disabling administrator requirements for secured methods, such as saving configuration of a software grabber. Or an alternative approach when the HyperHDR application is restarted remotely when you want to change the HDR settings: two full db HyperHDR configuration files required, one with tone mapping enabled and the other disable. One of the two paths must be passed as a parameter when starting HyperHDR to use the selected configuration.

**Reply by Titanos** [reply]
Allright, appreciate the answer! Is there a chance that the software grabber tone mapping will be controlled by the global HDR toggle in the future?

**Reply by awawa-dev** [reply]
Sure. You can request such a feature here: https://github.com/awawa-dev/HyperHDR/issues so I don't forget about it in the future

**Reply by Titanos** [reply]
I have now submitted the request. Thanks again!
